pkg/resources/grant_application_role_acceptance_test.go

```diff
@@ -0,0 +1,133 @@
+package resources_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/provider"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+
+	"github.com/hashicorp/terraform-plugin-testing/config"
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+)
+
+func TestAcc_GrantApplicationRole_accountRole(t *testing.T) {
+	applicationName := "int_test_app"
```

Comment:
This app is not created anywhere, so this test is missing the setup? Or am I missing something?

Reply:
They're created in Terraform config in https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/grant-app-role/pkg/resources/testdata/TestAcc_GrantApplicationRole/account_role/test.tf and https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/grant-app-role/pkg/resources/testdata/TestAcc_GrantApplicationRole/application/test.tf for each test case.

Reply:
They weren't before, closing.